.github/workflows/automationTemplate.yaml

```diff
@@ -85,46 +75,85 @@ jobs:
       
       - name: Install jq
         run: |
+          if [ ${{ env.CANCEL_WORKFLOW }} = true ]; then
```

Comment:
Tried using github [expressions ](https://docs.github.com/en/actions/learn-github-actions/expressions) but was unable to when referencing the environment variable: 
https://github.com/microsoft/Oryx/pull/1786/commits/e27d8f55d64176059d5fd5e789d103faff58e837#diff-3ed663dcfb84882e18b67fce2ab60dcb67f0b57ac69932ca286dcea41b5d80a5R83

Reply:
Tried using github [expressions ](https://docs.github.com/en/actions/learn-github-actions/expressions)but was unable to when referencing the environment variable:
https://github.com/microsoft/Oryx/commit/e27d8f55d64176059d5fd5e789d103faff58e837#diff-3ed663dcfb84882e18b67fce2ab60dcb67f0b57ac69932ca286dcea41b5d80a5R83

A startup failure would occur:
![image](https://user-images.githubusercontent.com/86327553/210661192-b8361fbc-5b80-4de7-8199-5fe00a2aa7c0.png)
https://github.com/microsoft/Oryx/actions/runs/3840383559